CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+<!-- omit in toc -->
+# Contribution Guide
+Welcome to the Open Network Edge Services Software (OpenNESS) project. OpenNESS is an open-source solution that is enriched by people like — you. Your contributions drive the network & enterprise edge computing!
+
+The rest of this document consists of the following sections:
+
+- [Code of Conduct](#code-of-conduct)
+- [Maintainers](#maintainers)
+- [Technical Steering Committee](#technical-steering-committee)
+- [Submitting Changes](#submitting-changes)
+- [Contribution Acceptance Flow](#contribution-acceptance-flow)
+- [How to report an issue/bug/enhancement](#how-to-report-an-issuebugenhancement)
+- [OpenNESS Release Schedule](#openness-release-schedule)
+- [Resources](#resources)
+- [Style Guide / Coding conventions](#style-guide--coding-conventions)
+- [Where can I ask for help?](#where-can-i-ask-for-help)
+- [License](#license)
+
+## Code of Conduct
+We at the OpenNESS community adhere to [Contributor Covenant](https://www.contributor-covenant.org/) as our Code of Conduct, and we expect project participants to adhere to it. Please read [the full text](CODE_OF_CONDUCT.md) so that you can understand what actions will and will not be tolerated.
+
+Instances of abusive, harassing or otherwise unacceptable behavior should be reported by contacting info@mail.openness.org.
+
+## Maintainers
+Maintainers act as the gatekeeper to the code - ensuring that coding standards, quality, and functionality of code are maintained by the contributors. Any additions to the source code must be approved by the appropriate maintainer before they are added. The maintainers' role include:
+
+* Ensuring that any code submitted is of the appropriate quality.
+* Ensuring that any code submitted is accompanied with appropriate documentation.
+* Ensuring that any code submitted has been reviewed by peers.
+* Ensuring that the relevant documentation is kept up to date with the code.
+* Ensuring that any identified bugs in the relevant code are captured in the issues backlog.
+* Ensuring that the unit, integration, and regression tests are appropriate for the relevant components.
+* Ensuring that the contribution does not infringe others' Intellectual Property or the appropriate license and that it is in compliance with [Developer Certificate of Origin](http://developercertificate.org/).
+* Answering questions/emails on the relevant areas of the OpenNESS code base.
+
+## Technical Steering Committee
```

Comment:
Do we need to say anything about the OpenNESS publishing the roadmap?

Reply:
Roadmap will be required if we have releasing schedule. For edgeapps there is no release schedule is defined.